weave/wandb_interface/wandb_stream_table.py

```diff
@@ -235,20 +179,20 @@ def __init__(
         self,
         table_name: str,
         *,
+        config: typing.Optional[dict] = None,
         project_name: typing.Optional[str] = None,
         entity_name: typing.Optional[str] = None,
-        _disable_async_file_stream: bool = False,
+        hidden: bool = True,
     ):
         super().__init__(
             table_name=table_name,
             project_name=project_name,
             entity_name=entity_name,
-            _disable_async_file_stream=_disable_async_file_stream,
+            config=config,
+            hidden=hidden,
         )
 
         self.queue: queue.Queue = queue.Queue()
-        atexit.register(self._at_exit)
```

Comment:
The `_at_exit` hook is already on StreamTableSync, no need for two of them.

Reply:
👍

Reply:
👍